src/components/CompareResults/TableHeader.tsx

```diff
@@ -67,19 +81,22 @@ function FilterableColumn({
             theme.palette.mode == 'light'
               ? Colors.Background200
               : Colors.Background200Dark,
-          borderRadius: '4px',
+          borderRadius: 0.5,
           fontSize: 'inherit',
+          padding: '6px 12px',
         })}
       >
-        {name}
         <FilterListIcon
           fontSize='small'
           color={hasFilteredValues ? 'primary' : 'inherit'}
-          sx={{ marginInlineStart: 1 }}
+          sx={{ marginInlineStart: 1, marginRight: 0.5, marginLeft: 0 }}
           titleAccess={
             hasFilteredValues ? 'Some filters are active' : 'No active filters'
           }
         />
+        {name}
+        <div className={filterCountStyle}>({possibleCheckedValues})</div>
```

Comment:
If you use Box instead of div, you could use `sx` here

Reply:
I don't mind doing that. I suppose we should transition out of `typestyle` soon for future styles development? At least have a consensus surrounding the topic. Maybe lets bring it up in next sync when we're all here.

Reply:
here I was more concerned about the consistency in this same component :)

but otherwise yeah we should make a decision and write it down

Reply:
I understand. Sounds good!